zanshinsdk/test_client.py

```diff
@@ -1051,6 +1051,23 @@ def test_get_organization_scan_target_scan(self):
             "GET", f"/organizations/{organization_id}/scantargets/{scan_target_id}/scans/{scan_id}"
         )
 
+
+    ###################################################
+    # Organization Scan Target Groups
+    ###################################################
+
+    def test_iter_organization_scan_target_groups(self):
```

Comment:
This is for sure the most important test. But I think we should have unit tests for input validation as well. For example ensuring we get a TypeError if we send anything other than a UUID or str as the organization ID, or a ValueError if we send a string that does not match a valid UUID.